CMakeLists.txt

```diff
@@ -571,9 +575,10 @@ if(H3_IS_ROOT_PROJECT AND BUILD_TESTING)
     add_h3_test(testGridDisksUnsafe src/apps/testapps/testGridDisksUnsafe.c)
     add_h3_test(testCellToParent src/apps/testapps/testCellToParent.c)
     add_h3_test(testCellToCenterChild src/apps/testapps/testCellToCenterChild.c)
+    add_h3_test(testCellToChildren_new src/apps/testapps/testCellToChildren_new.c)
     add_h3_test(testCellToChildren src/apps/testapps/testCellToChildren.c)
```

Comment:
Wasn't the plan to delete the old one and rename `_new` to the old name?

Reply:
Yes. Just haven't gotten to it yet. As long as people are comfortable that the new code passes the old tests, I can delete it. :)

Reply:
It's been run multiple times in this PR, correct? I say delete it.

Reply:
Yup. Taking it out!